### ISSUE TITLE
Provide request matcher for error path

### ISSUE THREAD

**Issue by mbhave** [spring-projects/spring-boot#10958]
This can be used in custom security configuration (similar to `StaticResourceRequest`)

**Comment by mbhave** [comment]
For webflux as well

**Comment by snicoll** [comment]
@mbhave why is this issue blocked?

**Comment by mbhave** [comment]
you're right. Only the webflux part of it is blocked on https://jira.spring.io/browse/SPR-16298

**Comment by mbhave** [comment]
@snicoll probably a separate issue, but I'm not sure if the error path can be configured for webflux atm.

**Comment by mbhave** [comment]
For servlets, it doesn't seem like the forward to the error path goes through the entire Spring Security filter chain so providing a RequestMatcher for it would be pointless. 

@bclozel mentioned that for webflux errors are handled in all cases using the current path and no HTTP redirection nor re-dispatch is involved. 

We can reopen the issue if anyone has any concerns.
